extern/CMakeLists.txt

```diff
@@ -1,5 +1,231 @@
 include(Macros)
 
+if(MSVC)
```

Comment:
What if we set `CESIUM_NATIVE_DEPENDENCIES` here to include the common dependencies, then append the platform-specific ones? If we also do this in `src/core/CMakeLists.txt`, that cuts this list down from four occurrences to two.

Reply:
I'll see if we can do that, with the caveat that dependency order sometimes matters.

Reply:
Fixed in https://github.com/CesiumGS/cesium-omniverse/pull/729/commits/1aaf653a10e77fe572ff4f948facfc57ab024051 with a slightly different approach